README.md

```diff
@@ -9,3 +10,17 @@ might help with the notation for mathematical expressions.
 $G_1=(V_1 , E_1)$ is isomorphic to $G_2 = (V_2, E_2)$ if there exists a
 one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,v)
 \in E_1$ iff $(f(u),f(v)) \in E_2$.
+
+## Proof
+
+We can prove this through contridiction
+
+Let $A$ and $B$ both be graphs, and $A$ = $(V_1, E_1)$, $B$ = $(V_2, E_2)$
+Say that $A$ continas fewer nodes than $B$.
+Let $f$ be the bijective transformation $f: V_1 \rightarrow V_2$ such that $(u,v)
+\in E_1$ iff $(f(u),f(v)) \in E_2$, since this function is bijective it is both one to one and onto.
+Then if we take the map of $V_1 \rightarrow V_2$ we can see that there will be shared solutions as $A$ contains fewer nodes than $B$.
```

Comment:
What are "shared solutions"? Solutions to what?

Reply:
by that I meant that because one graph has fewer nodes than the other, the function would map to multiple solutions, instead of one node to one node, which is what violates a function being one to one.

Reply:
Not necessarily -- there could be nodes in $B$ that are not mapped. But please use this terminology -- the function is mapping nodes to one another; it's not solving anything.

Reply:
wouldn't unmapped nodes still violate a one to one mapping?

Reply:
Yes. You just have to consider all the possibilities.